docs/how-to-setup-freecodecamp-locally.md

```diff
@@ -160,7 +160,7 @@ If you are on a different OS and/or are still running into issues, reach out to
 ### Configuring dependencies
 
 #### Step 1: Setup the environment variable file
-The default API keys and environment variables are stored in the file `sample.env`.  This file needs to be copied to a new file named `.env` that is accessed dynamically during the installation step.
+Within the root directory `freeCodeCamp`, the default API keys and environment variables are stored in the file `sample.env`. This file needs to be copied to a new file named `.env` that is accessed dynamically during the installation step. From here moving forward, all commands should be run from the root directory.
```

Comment:
```suggestion
The default API keys and environment variables are stored in the file `sample.env`, found in the root directory, `freeCodeCamp`. This file needs to be copied to a new file named `.env` that is accessed dynamically during the installation step. From here moving forward, all commands should be run from the root directory.
```
The previous wording sounded a little odd - as if 'within the root directory' the keys are stored in `sample.env`, but outside the root directory they're stored somewhere else.  For example "Within London beer is stored in casks" implies that outside of London it isn't.

Possibly I'm reading too much into this!